packages/kujua-sms/views/lib/smsparser.js

```diff
@@ -1,4 +1,5 @@
 var utils = require('kujua-utils'),
+    standardiseDigits = require('eurodig'),
```

Comment:
I'm not 100% I haven't tried it, but you need a special path "views/lib/eurodig" for couchdb to resolve `require` properly.  If this passes tests then either it works fine or our tests on not testing the backend very well.